src/core/qgscoordinatereferencesystem.cpp

```diff
@@ -333,15 +333,20 @@ bool QgsCoordinateReferenceSystem::createFromOgcWmsCrs( const QString &crs )
 
   QString wmsCrs = crs;
 
-  QRegExp re( "urn:ogc:def:crs:([^:]+).+([^:]+)", Qt::CaseInsensitive );
-  if ( re.exactMatch( wmsCrs ) )
+  QRegExp reUri( "http://www\\.opengis\\.net/def/crs/([^/]+).+/(\\d+)", Qt::CaseInsensitive );
```

Comment:
You may want to use raw strings here for readability (avoid double slashes):

Reply:
@elpaso -sorry for the late responce. Were you thinking of something like this?
`QRegExp reUri( R"(http://www\.opengis\.net/def/crs/([^/]+).+/(\d+))", Qt::CaseInsensitive );`

Reply:
Yes, that should work (untested)